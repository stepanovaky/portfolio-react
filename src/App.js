import React from 'react';
import ProjectsIndex from './Projects/ProjectsIndex';
import { Switch, Route, Link } from 'react-router-dom';



function App() {
  return (
    <div className="App">
      <ProjectsIndex />
      




      
    </div>
  );
}

export default App;
