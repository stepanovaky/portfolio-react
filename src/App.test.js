import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import './setupTests';

it('renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})

it('renders without crashing', () => {
  shallow(<App />);
})