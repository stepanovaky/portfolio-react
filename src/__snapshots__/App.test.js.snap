// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="projects-index"
  >
    <ul>
      <li>
        <div
          className="king-henry-quiz"
        >
          <h2>
            King Henry VIII Quiz App
          </h2>
        </div>
      </li>
      <li>
        <div
          className="to-do-list"
        >
          <h2>
            To Do List
          </h2>
        </div>
      </li>
    </ul>
  </div>
  u
</div>
`;
